### PR TITLE
genpolicy-msft: bump version to 3.2.0.azl1.genpolicy0

### PR DESCRIPTION
**Pull request by burgerdev**
This release is what's currently available on AKS.

### CODE REVIEW

File: packages/by-name/genpolicy-msft/package.nix [hunk 1]
```diff
@@ -60,11 +46,15 @@ rustPlatform.buildRustPackage rec {
...
+  preBuild = ''
+    make src/version.rs
+  '';
+
   passthru = rec {
     settings = fetchurl {
       name = "${pname}-${version}-settings";
-      url = "https://github.com/microsoft/kata-containers/releases/download/${version}/genpolicy-settings.json";
-      hash = "sha256-Q19H7Oj8c7SlPyib96fSRZhx/nJ96HXb8dfb9Y/Rsw8=";
+      url = "https://raw.githubusercontent.com/microsoft/kata-containers/${version}/src/tools/genpolicy/genpolicy-settings.json";
```
Comment (katexochen) [inline packages/by-name/genpolicy-msft/package.nix:56]: Should we add a comment to switch this back to the release artifact once they publish the next genpolicy relelease?

**Review by katexochen — APPROVED** [review]

File: packages/by-name/genpolicy-msft/package.nix (hunk 1 above)
Comment (burgerdev) [reply packages/by-name/genpolicy-msft/package.nix:56]: Given that the tag is out for 3 weeks by now, and that `3.2.0.azl1.genpolicy0` and `3.2.0.azl1` track the same commit, I don't expect them to publish these files as release artifacts.

Comment (katexochen) [reply packages/by-name/genpolicy-msft/package.nix:56]: Maybe they forgot this time?

Comment (burgerdev) [reply packages/by-name/genpolicy-msft/package.nix:55]: Comment added

**Pull request merged**